api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
```

Comment:
Yes, of course, we will have it. 
But I just wanted to describe the subscription creation endpoint - what we already discussed and agreed on.